custom_components/entsoe/const.py

```diff
@@ -28,6 +30,7 @@
 }
 
 ENERGY_SCALES = { "kWh": 1000, "MWh": 1 }
+PERIOD_OPTIONS = ["PT60M", "PT15M"]
```

Comment:
I think it would be better to have something like "native" too. Hence, the raw input, versus aggregated options.